package.json

```diff
@@ -97,6 +102,7 @@
     "karma-sinon": "1.0.4",
     "mocha": "1.21.4",
     "sinon": "1.10.3",
+    "through2": "^2.0.3",
```

Comment:
good catch. these things were just auto-generated from the install command, and i forgot to check on it